shynet/dashboard/templates/dashboard/pages/service.html

```diff
@@ -109,8 +109,24 @@
             <tbody>
                 {% for location in stats.locations %}
                 <tr>
-                    <td class="truncate w-full max-w-0">{{location.location|default:"Unknown"|urldisplay}}</td>
-                    <td class="rf">{{location.count|intcomma}}</td>
+                    <td class="truncate w-full max-w-0 relative">
+                        <div
+                            class="absolute h-6"
+                            style="width: {{location.count|percent:stats.hit_count}}; top: 8px; left: 8px; background-color: var(--color-urge-200-fallback)"
+                        >
+                        </div>
+                        <div class="absolute" style="top: 8px; left: 8px">
```

Comment:
replace `class="absolute" style="top: 8px; left: 8px"` with `class="relative"` for same effect

Same thing for the other tables